userguide/kaui/includes/users.adoc

```diff
@@ -0,0 +1,165 @@
+
+* https://killbill.io/blog/multi-tenancy-authorization/[Multi-Tenancy and Authorization]
+
+* https://github.com/killbill/killbill-api/blob/master/src/main/java/org/killbill/billing/security/Permission.java[List of user permissions]
+
+=== Managing Users with a Third-Party System
+
+If you are managing users with a third-party system, all that's required in Kaui is to add the username per the <<Add a User>> section.
+
+=== Managing Users with Kill Bill
+
+If you are managing users with the Kill Bill database _or_ solely with the `shiro.ini` file, first define the users and roles as described in https://docs.killbill.io/latest/user_management.html[Users, Roles, and Permissions Management]. Then, in Kaui, you must create the same users, passwords, etc. For more information, see <<Add a User>> and <<Edit a User>>.
+
+// ADD "Permissions" and "User Role" to Glossary
+// Permissions:: A list of operations that can be performed in the system (for example, `CREATE_USER` or `INVOICE_CAN_CREDIT`). A Kill Bill or Kaui administrator can assign permissions to roles, which are then assigned to users to control users' access to the system.
+
+== Before You Begin
+
+If your organization uses a third-party system, such as LDAP, Okta, or Auth0, to manage Kill Bill users and roles, first configure them in that system, then add them in Kaui. For more information, see https://docs.killbill.io/latest/user_management.html[Users, Roles, and Permissions Management].
+
+If you are using the Kill Bill database to store user data, adding a user profile to in Kaui automatically creates it in Kill Bill. **NOT SURE ABOUT THIS
+
+== Create a New User Role
+
+The role(s) assigned to a user define what tasks the user can perform in the system (for example, perform a chargeback or add a custom field).
+
+. Open the Add New Role Definition screen, either from the User Update screen (see <<Edit a User>>) or by accessing `https://demo.killbill.io/role_definitions/new`.
++
+image::AddNewRole.png[width=450,align="center"]
++
+. Enter the *Role name* for the user. Do not use any spaces in the role name.
+. Enter all the *Permissions* that are associated with this role. See the https://github.com/killbill/killbill-api/blob/master/src/main/java/org/killbill/billing/security/Permission.java[list of Kill Bill user permissions] as a reference.
+. Click the *Save* button.
+
+=== Add a User
+
+. On the homepage, click the gears icon at the top of the screen.
++
+image::Homepage-SettingsDropdown.png[width=650,align="center"]
++
+. Select *Users.*
++
+Kaui displays [SOMETHING]. _I don't have access to this screen._
++
+. Select [SOMETHING ELSE].
++
+Kaui displays the Add New User pop-up:
++
+image::AddNewUser.png[width=450,align="center"]
++
+[NOTE]
+*Note:* You can also open the Add New User screen by clicking on the username and tenant name in the upper right corner and clicking the plus icon ( image:i_PlusGreen.png[] ) next to "Allowed Users."
++
```

Comment:
Are you sure about this? In my local installation, the `+` icon next to `Allowed Users` displays a screen that only allows adding a username. (Please see the attached screenshot)
<img width="457" alt="add-allowed-user" src="https://user-images.githubusercontent.com/85998496/152750413-976470df-d27b-4509-8971-361101545296.png">

Reply:
Actually, that's what happened to me, but I (erroneously) thought that my permissions were restricting me from seeing all of the user fields (password, roles, etc.) at once. I will correct.

Reply:
@pierre or @sbrossie - Why would this screen be used to only create a user name?

Reply:
See my comment before related to `Delete User`. I think deleting a user from the `Tenant Screen` (the screen that is displayed on clicking username and tenant name in the upper right corner) only disassociates the user from the tenant. You can link a user back to a tenant using the above `Add Allowed User` screen, but again, this is my theory, I will let the experts confirm 😛

Reply:
This is to allow whomever is identified by this username to access the current tenant.

Reply:
Ah! Now I finally understand!